Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLSchemaBuilder.swift

```diff
@@ -0,0 +1,248 @@
+//
+// Created by Sadik Ekin Ozbay on 06.01.21.
+//
+
+@_implementationOnly import Vapor
+import GraphQL
+
+
+func graphqlTypeMap(with type: Codable.Type) -> GraphQLScalarType {
+    if (type == String.self) {
+        return GraphQLString
+    } else if (type == Int.self || type == UInt.self) {
+        return GraphQLInt
+    } else if (type == Float.self) {
+        return GraphQLFloat
+    } else if (type == Bool.self) {
+        return GraphQLBoolean
+    }
+    return GraphQLString
```

Comment:
I'm not so sure about this? I feel like we could end up marking a lot of types as Strings and not even notice until they crash.
It's always best to fail as early as you can tell somethings wrong